src/kirby/components/modal/alert/alert.component.ts

```diff
@@ -11,8 +12,29 @@ export class AlertComponent implements AfterViewInit {
   @ViewChild('alertWrapper', { static: true }) private alertWrapper: ElementRef;
   private scrollY: number = Math.abs(window.scrollY);
 
-  @Input() title: string;
-  @Input() message: string;
+  private _title: string & Observable<string>;
```

Comment:
Why do all this? Why not just:
`@Input() title: string || Observable<string;`

Reply:
Because then you can only get the union and can't subscribe to it.